framework/source/class/qx/ui/basic/Image.js

```diff
@@ -764,7 +769,7 @@ qx.Class.define("qx.ui.basic.Image",
           if (qx.core.Environment.get("qx.debug")) {
             this.assertObject(font, "Virtual image source contains unkown font descriptor");
           }
-          size = font.getSize();
+          size = parseInt(source.split("/")[2] || font.getSize());
```

Comment:
radix

Reply:
Adjusted these to have a radix, too. Thanks for your feedback.